src/components/slide/selection-frame.tsx

```diff
@@ -221,48 +205,83 @@ export const SelectionFrame: React.FC<Props> = ({ children, treeId }) => {
 
   const isHovered = hoveredElementId === children.props.id;
   const isSelected = editableElementId === children.props.id;
-  const isSelectedAndMarkdown = isSelected && isMdElement(selectedElement);
+
+  const handleMouseDown = useCallback(
+    (event: MouseEvent<HTMLDivElement>) => {
+      if (
+        (event.target as HTMLElement).classList.contains('moveable-control')
+      ) {
+        return;
+      }
+      event.stopPropagation();
+      dispatch(deckSlice.actions.editableElementSelected(children.props.id));
+
+      if (event.detail === 2) {
+        setDoubleClickedElement(true);
```

Comment:
Looking at PowerPoint and Google Slides, both of them let a single click in a textbox trigger the editing mode. Moving around and resizing is accomplished by clicking the textbox closer to the outer edge and dragging. Should we go with that behavior instead, since casual users will probably be most familiar with that?

Reply:
Checking Keynote either double clicking into edit or clicking once after being selected works. That's nuanced since a click down in the web and mouse down for drag is viewed as the same. Frame movement would need to be customized using React DND if we wanted to go that route, but it's a great suggestion.